templates/search/results-no-results-fr.html

```diff
@@ -0,0 +1,46 @@
+---
+{
+	"title": "Résultats de la recherche (aucun résultat)",
+	"language": "fr",
+	"altLangPage": "results-no-results-en.html",
+	"pageType": "recherche",
+	"pageclass": "page-type-search",
+	"dateModified": "2025-07-25",
+	"share": "false",
+	"deptfeature": false,
+	"nositesearch": true
+}
+---
+<form role="search">
+	<label class="wb-inv" for="sch-inp">Recherchez le site Web du Gouvernement du Canada</label>
+	<div class="input-group mrgn-tp-lg">
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query name="q" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+			<button type="submit" class="btn btn-primary"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Recherche</span></button>
+		</span>
+	</div>
+	<p id="gc-pi" class="mrgn-tp-md">N'incluez pas de renseignements personnels (téléphone, courriel, NAS, renseignements financiers, médicaux ou professionnels).</p>
+</form>
+
+<div data-gc-search="{}">
+	<div id="wb-land">
```

Comment:
This should be a section

```suggestion
	<section id="wb-land">
```